source/kernel/src/forth/mod.rs

```diff
@@ -212,16 +254,25 @@ impl<'forth> AsyncBuiltins<'forth, MnemosContext> for Dispatcher {
 }
 
 impl Params {
+    const DEFAULT_STACK_SIZE: usize = 256;
+    const DEFAULT_INPUT_BUF_SIZE: usize = 256;
+    const DEFAULT_OUTPUT_BUF_SIZE: usize = 256;
+    const DEFAULT_DICTIONARY_SIZE: usize = 4096;
+    const DEFAULT_STDIN_CAPACITY: usize = 1024;
+    const DEFAULT_STDOUT_CAPACITY: usize = 1024;
+    const DEFAULT_BAG_OF_HOLDING_CAPACITY: usize = 16;
+    const DEFAULT_SPAWNULATOR_TIMEOUT: Duration = Duration::from_secs(5);
```

Comment:
personally, i like making the default values `pub`, so that downstream code can say stuff like 

```rust
Params {
    stack_size: Params::DEFAULT_STACK_SIZE * 2, // double the default stack size
   ...Default::default()
}
```
or whatever. not a big deal though.